proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -35,6 +35,45 @@ message BreakingChangePolicy {
   BreakingChangeCategory category = 3;
 }
 
+// UniquenessPolicy is the policy for uniqueness on the server.
+message UniquenessPolicy {
+  // Whether uniqueness policy is enforced.
+  bool enabled = 1;
+}
+
+// UniquenessState is the state of collisions on the server.
+message UniquenessState {
+  // The existing collisions on the server.
+  repeated Collision collisions = 1;
+}
+
+// Collision is a collision that has occurred.
+message Collision {
+  // The reference that has collisions.
+  string reference = 1;
+  // The type of collision.
+  CollisionType type = 2;
+  // The sources of the collision.
+  repeated CollisionSource sources = 3;
+}
+
+// CollisionSource is the source of a collision
+message CollisionSource {
+  string owner_id = 1;
+  string repository_id = 2;
+  string path = 3;
+}
```

Comment:
I'm not sure it's worth structuring the collision like this for the admin service.

I think we might be better off just building some informative error(s) in the application and returning them.

e.g. 

```
// UniquenessState is the state of collisions on the server.
message UniquenessState {
  // The existing collisions on the server.
  repeated string collisions = 1;
}
```

Where a collision is something like:
```
Fully qualified name "foo.Bar.Baz" is found { N } times: 
 - {owner}/{repository} in foo/bar/baz.proto
 - {owner}/{repository} in types/v1beta1/baz.proto
 ```
 ```
Proto file path "types/v1alpha1/datetime.proto" is found { N } times:
- {owner}/{repository}
- {owner}/{repository}
- {owner}/{repository}
```
 
@stefanvanburen, @rebello95  What do you think? This would be different to what users might see on push.

Reply:
I like the format of those errors, but I agree with @stefanvanburen that the API should be structured and allow the frontend decide how to show the info